recipes/cluster-picker/run_test.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+echo -e ">seq1\nATTA\n>seq2\nATTA\n>seq3\nATTA\n>seq4\nATTA" > dummy.fasta
```

Comment:
You could add these 3 lines also to the commands-test section into meta.yaml. Cleanup is not needed.